lib/src/util/safe_render_manager/safe_render_manager.dart

```diff
@@ -52,20 +52,35 @@ class SafeRenderManager extends Disposable {
   /// If not specified, a new div will be used.
   final Element mountNode;
 
+  final _contentRefObject = createRef<dynamic>();
+
   /// The ref to the component rendered by [render].
   ///
   /// Due to react_dom.render calls not being guaranteed to be synchronous.
   /// this may not be populated until later than expected.
-  dynamic contentRef;
+  dynamic get contentRef => _contentRefObject.current;
```

Comment:
This is technically a breaking change since it removes a setter, but setting `contentRef` shouldn't have ever been supported, and I can't find any code that sets this.

Reply:
Actually, I decided to avoid a breakage here ee8ab08a8c48308b9218605ce62c1b15544b66c3